ee/models/license.py

```diff
@@ -61,8 +61,6 @@ class License(models.Model):
         AvailableFeature.PATHS_ADVANCED,
         AvailableFeature.CORRELATION_ANALYSIS,
         AvailableFeature.GROUP_ANALYTICS,
-        AvailableFeature.MULTIVARIATE_FLAGS,
```

Comment:
err, not sure if this might have some backwards compat issues with billing? Can @raquelmsmith confirm?

I don't think it matters _to us_ anymore whether this is here or not, since we're not checking for this feature availability anywhere anymore

Reply:
If it's not referenced anywhere then it's fine to remove!